README.md

```diff
@@ -1,8 +1,72 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# JJ Falcon . Jarvis Consulting
 
+ I completed my bachelor's degree in Computer Science at Ryerson University with a software engineering concentration in December 2020. The concentration allowed me to focus my academics and excelled on the entire Software Development cycle. It also exposed me to different Data Management Methodologies. Throughout my coop experiences, I have gained a great understanding and hands-on experience of the software development life cycle. This includes business requirements gathering,  designing solutions,  implementing and maintaining the solution in order to produce results that are meaningful to the organization. Additionally, through the Jarvis training program, I have had the opportunity to use additional technologies in manipulating and mining data. I am a quick learner, analytical thinker with a customer-centric approach to my work ethic. I am a firm believer that success at work means helping others be better at what they do. There is no greater motivation for me than to see my work or be a part of a team whose work has a direct impact on making other people's life easier. This is what excites me about the software industry. 
```

Comment:
I completed my bachelor's degree in Computer Science at Ryerson University with a software engineering concentration in December 2020. The concentration allowed me to focus my academics and excelled on the entire Software Development cycle. It also exposed me to different Data Management Methodologies. I have gained a great understanding and hands-on experience of the software development life cycle throughout my coop experiences. This includes business requirements gathering, designing solutions, implementing and maintaining the solution to produce meaningful results to the organization. Additionally, through the Jarvis training program, I have had the opportunity to use additional technologies in manipulating and mining data. I am a quick learner, analytical thinker with a customer-centric approach to my work ethic. I am a firm believer that success at work means helping others be better at what they do. There is no greater motivation for me than to see my work or be a part of a team whose work directly impacts making other people's lives easier. This is what excites me about the software industry. 

fix some sentences